react/components/ComponentEditor/index.tsx

```diff
@@ -834,14 +853,15 @@ export default compose(
   graphql(ExtensionConfigurations, {
     name: 'extensionConfigurations',
     options: ({
-      editor: { editTreePath, iframeWindow },
-      runtime: { extensions, page },
+      editor: { editTreePath },
+      runtime: { context, extensions, page },
     }: ComponentEditorProps) => ({
+      fetchPolicy: 'network-only',
       variables: {
-        configurationsIds: extensions[editTreePath as string].configurationsIds,
+        configurationsIds: extensions[editTreePath!].configurationsIds,
```

Comment:
Just got a `TypeError: Cannot read property 'configurationsIds' of undefined` here 👀